pkg/list/list_vendor.go

```diff
@@ -0,0 +1,217 @@
+package list
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/cloudposse/atmos/internal/exec"
+	"github.com/cloudposse/atmos/pkg/config"
+	"github.com/cloudposse/atmos/pkg/schema"
+	"github.com/cloudposse/atmos/pkg/ui/theme"
+	"github.com/cloudposse/atmos/pkg/utils"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	DefaultCSVDelimiter = ","
+	DefaultTSVDelimiter = "\t"
+)
+
+// VendorInfo represents a vendor configuration entry
+type VendorInfo struct {
+	Component string `json:"component"`
+	Type      string `json:"type"`
+	Manifest  string `json:"manifest"`
+	Folder    string `json:"folder"`
+	Version   string `json:"version"`
+}
+
+// processVendorFile processes a vendor configuration file and returns vendor information
```

Comment:
The fact we are not using the vendoring logic that we have today, is concerning. We support imports, directories, globbing, templates, deep merging etc. We cannot reimplement this here.